applications/nrf5340_audio/index.rst

```diff
@@ -8,6 +8,32 @@ The nRF5340 Audio applications demonstrate audio playback over isochronous chann
 .. note::
    nRF5340 Audio applications and their DFU/FOTA functionality are marked as :ref:`experimental <software_maturity>`.
 
+The following table summarizes the differences between the available nRF5340 Audio applications.
+
+.. list-table:: Differences between nRF5340 Audio applications
+   :header-rows: 1
+
+   * - Application
+     - LE Audio mode and role
+     - Minimum amount of nRF5340 Audio DKs recommended for testing
+     - FEM support
+   * - :ref:`Unicast client<nrf53_audio_unicast_client_app>`
+     - CIS gateway
+     - 2
+     - ✔
+   * - :ref:`Unicast server<nrf53_audio_unicast_server_app>`
+     - CIS headset
+     - 3
+     - ✔
+   * - :ref:`Broadcast sink<nrf53_audio_broadcast_sink_app>`
+     - BIS gateway
+     - 2
+     -
+   * - :ref:`Broadcast source<nrf53_audio_broadcast_source_app>`
+     - BIS headset
+     - 2
+     - ✔
```

Comment:
@alexsven , @koffes , @Krrrl , anything else you would add to this table?

Reply:
It would make sense to rather point out that you need 2-3 DKs to be able to test out unicast, and you need unicast_client and unicast_server applications
In all cases it makes most sense to have 3 DKs, one gateway and two headsets/earbuds

Reply:
Changed to 3 for both unicasts.